src/view/com/composer/Composer.tsx

```diff
@@ -331,14 +333,14 @@ export const ComposePost = observer(function ComposePost({
                     end={{x: 1, y: 1}}
                     style={styles.postBtn}>
                     <Text style={[s.white, s.f16, s.bold]}>
-                      {replyTo ? 'Reply' : 'Post'}
+                      {replyTo ? _(msg`Reply`) : _(msg`Post (verb)`)}
                     </Text>
                   </LinearGradient>
                 </TouchableOpacity>
               ) : (
                 <View style={[styles.postBtn, pal.btn]}>
                   <Text style={[pal.textLight, s.f16, s.bold]}>
-                    <Trans>Post</Trans>
+                    <Trans>Post (verb)</Trans>
```

Comment:
Why was `(verb)` added here?

Reply:
In English, they are both same "Post", but some other languages use different noun and verb.

I've added `(verb)` to distinguish between them, but if there's another way to do it, I'd like to change it.

Reply:
In English, they both appear as "Post", just like they always have.

Reply:
As mentioned in #2446, this appears to be an issue with other languages as well, including Portuguese. I think it needs to add `(verb)` or some other text after the `msgid` to distinguish them.